api/scpca_portal/metadata_file.py

```diff
@@ -131,4 +131,6 @@ def format_metadata_dict(metadata_dict: Dict) -> Dict:
     """
     Returns a copy of metadata dict that is formatted and ready to be written to file.
     """
+    # Replace empty blank or None values with "NA"
+    metadata_dict.update({k: common.NA for k, v in metadata_dict.items() if v in ["", None]})
```

Comment:
Let's not mutate the argument here.
Is this line added because there are empty / None values in the test data?